src/middleware/authentication.ts

```diff
@@ -11,10 +12,18 @@ export const isAuthenticated = (
   passport.authenticate(
     'jwt',
     { session: false },
-    (err: Error, user: UserAttributes) => {
+    async (err: Error, user: UserAttributes) => {
       if (!user) {
         return sendResponse(res, 403, null, 'You are not authorized');
       }
+      const authorization = req.header('Authorization')?.split(' ')[1];
+
+      const logout = await Database.Logout.findOne({
+        where: { token: authorization },
+      });
+      if (logout) {
+        return sendResponse(res, 403, null, 'not authorized');
+      }
```

Comment:
and again they told you to use function not implement it in here

Reply:
I understand when i use a function i will implement it where?